wherehows-frontend/app/controllers/Application.java

```diff
@@ -70,11 +78,41 @@ private static Result serveAsset(String path) {
         return ok(indexHtml).as("text/html");
     }
 
-    public static Result healthcheck()
-    {
+    public static Result healthcheck() {
         return ok("GOOD");
     }
 
+    public static Result deployLibsFrontend() {
+        String libPath = WHZ_APP_ENV + "/lib";
+        String commitFile = WHZ_APP_ENV + "/commit";
+        String libraries = "";
+        String commit = "";
+
+        try {
+
+            BufferedReader br = new BufferedReader(new FileReader(commitFile));
+            commit = br.readLine();
+
+        } catch (IOException ioe) {
+            Logger.error("Error while reading commit file. Error message: " +
+                    ioe.getMessage());
+        }
+
+        if (WHZ_APP_ENV == null) {
```

Comment:
This should be the first line of this function, also print some helpful message like "WHZ_APP_HOME environmental variable not defined"